src/libs/ReportUtils.js

```diff
@@ -411,16 +412,73 @@ function formatReportLastMessageText(lastMessageText) {
 }
 
 /**
- * Helper method to return a default avatar
+ * Helper method to return the default avatar associated with login
  *
  * @param {String} [login]
  * @returns {String}
  */
 function getDefaultAvatar(login = '') {
-    // There are 8 possible default avatars, so we choose which one this user has based
+    // There are 24 possible default avatars, so we choose which one this user has based
     // on a simple hash of their login
+    if (!login) {
+        return Expensicons.FallbackAvatar;
+    }
+    if (login === CONST.EMAIL.CONCIERGE) {
+        return Expensicons.ConciergeAvatar;
+    }
+
+    const loginHashBucket = (Math.abs(hashCode(login.toLowerCase())) % 24) + 1;
+
+    return defaultAvatars[`Avatar${loginHashBucket}`];
+}
+
+/**
+ * Helper method to return old dot default avatar associated with login
+ *
+ * @param {String} [login]
+ * @returns {String}
+ */
+function getOldDotDefaultAvatar(login = '') {
+    // There are 8 possible old dot default avatars, so we choose which one this user has based
+    // on a simple hash of their login
+    if (login === CONST.EMAIL.CONCIERGE) {
+        return CONST.CONCIERGE_ICON_URL;
+    }
     const loginHashBucket = (Math.abs(hashCode(login.toLowerCase())) % 8) + 1;
-    return `${CONST.CLOUDFRONT_URL}/images/avatars/avatar_${loginHashBucket}.png`;
+
+    return `${CONST.CLOUDFRONT_URL}/images/avatars/avatar_${loginHashBucket}`;
+}
+
+/**
+ * Returns true if provided URL requires a default avatar
+ * @param {String} [avatarURL]
+ * @returns {Boolean}
+ */
+function isDefaultAvatar(avatarURL) {
```

Comment:
I'm a bit confused what this function does just by the name & description - from the name `isDefaultAvatar` I'd guess we're checking if the passed URL is the URL of a default avatar, but the description says we're checking if a URL "requires a default avatar" - what does that mean? Maybe we can clarify :D